catchpoint-mixin/README.md

```diff
@@ -0,0 +1,94 @@
+# Catchpoint Mixin
+
+The Catchpoint mixin is a set of configurable Grafana dashboards and alerts.
+
+The Catchpoint mixin contains the following dashboards:
+
+- Catchpoint overview
+- Catchpoint web performance by test name
+- Catchpoint web performance by node name
+
+and the following alerts:
+
+- CatchpointHighDNSResolutionTime
+- CatchpointTotalTimeExceeded
+- CatchpointContentLoadingDelays
+- CatchpointHighServerResponseTime
+- CatchpointHighFailedRequestRatio
+
+## Catchpoint overview
+
+The Catchpoint overview dashboard provides details on alerts, load times, request success rates, connections, and errors.
+
+![Catchpoint overview dashboard]()
+
+## Catchpoint web performance by test name
+
+The Catchpoint web performance by test name dashboard provides details on page completion times, response sizes, network connections, requests, content types, and errors. These metrics are all grouped by test name.
+
+![Catchpoint web performance by test name]()
+
+## Catchpoint web performance by node name
```

Comment:
To Shahzeb's point, I don't think we need to specify "test name" and "node name", just drop the "name" part. Applies throughout the readme